src/components/Game.vue

```diff
@@ -401,6 +418,11 @@ export default {
         if (localStorage.getItem('keyboard')) {
             this.keyboard = JSON.parse(localStorage.getItem('keyboard'));
         }
+
+        // 👩
+        if (this.finished) {
```

Comment:
Pour éviter un bug d'affichage quand on sort du mode archive :
```suggestion
        if (this.finished && !this.archivesMode) {
```
<img width="520" alt="Capture d’écran 2022-03-06 à 12 32 46" src="https://user-images.githubusercontent.com/775771/156921190-90785301-502a-4aba-bb32-9887d1856179.png">